Source/Evolve/WarpXEvolveEM.cpp

```diff
@@ -61,25 +61,25 @@ WarpX::EvolveEM (int numsteps)
         if (warpx_py_beforestep) warpx_py_beforestep();
 #endif
 
-        if (costs[0] != nullptr)
-        {
+        MultiFab* cost = WarpX::getCosts(0);
+        amrex::Vector<amrex::Real>* cost_heuristic = WarpX::getCostsHeuristic(0);
+        if (cost != nullptr || cost_heuristic != nullptr) {
 #ifdef WARPX_USE_PSATD
             amrex::Abort("LoadBalance for PSATD: TODO");
 #endif
-
             if (step > 0 && (step+1) % load_balance_int == 0)
             {
                 LoadBalance();
-                // Reset the costs to 0
-                for (int lev = 0; lev <= finest_level; ++lev) {
-                    costs[lev]->setVal(0.0);
-                }
```

Comment:
@MaxThevenet I have removed this because `RemakeLevel` called in `LoadBalance` seems to do the same thing.  Is this really redundant or does it guard against a special case?  Please see `RemakeLevel` in WarpXRegrid.cpp..

Reply:
Thanks for pointing this out. It seems that you're right, but there could be some corner case for which this safeguard is useful. Maybe @WeiqunZhang or @atmyers would know. For safety, I would let it as it was, as it is not directly the topic of this PR.

Reply:
I've reverted to clearing levels here.